manim/mobject/types/vectorized_mobject.py

```diff
@@ -662,25 +669,25 @@ def gen_subpaths_from_points_2d(self, points):
     def get_subpaths(self):
         return self.get_subpaths_from_points(self.get_points())
 
-    def get_nth_curve_points(self, n):
+    def get_nth_curve_points(self, n: int) -> Sequence[Vector]:
```

Comment:
`self.points` is a list right? Then:
```suggestion
    def get_nth_curve_points(self, n: int) -> List[Vector]:
```